website/docs/advanced/targeting.md

```diff
@@ -194,6 +194,67 @@ The exact value that will be served to the users that fall into that fraction. D
 | Whole Number   | Integer      | any whole number within the range of `Int32`      |
 | Decimal Number | Double       | any decimal number within the range of `double` |
 
+## Stickiness & Consistency
+
+The percentage-based targeting is sticky by design and consistent across all SDKs.
+
+Percentage-based targeting is based on the identifier passed in the `User Object` to get `getValue()` methods in the SDKs. 
+The SDKs are hashing the identifier concatenated with the evaluated Feature Flag's key and assign a 0-99 number to the User for a specific Feature Flag. 
+This number is fix and consistent across all SDKs. The SDKs are checking if the assigned number is greater or less than the percentage set on the ConfigCat Dashboard.
+
+> It is important to note that not only the User's identifier is hashed but the User's identifier concatenated with the evaluated Feature Flag's key.
+With this concatenation we can ensure that you won't test on the same user base for all of your Feature Flags.
+
+> As the evaluation happens in the SDKs, your user's sensitive information will never leave your system. The data flow is unidirectional (only from ConfigCat CDN servers to your SDKs) and ConfigCat doesn't receive or store any of the values passed in the User Object to the SDKs.
```

Comment:
any of the User Object's attributes passed to the SDKs

Reply:
if it's not a warning I think it should be in an "info" box
:::info
text
:::

Reply:
Fixed